addons/web/static/src/core/datetime/datetime_picker.js

```diff
@@ -330,6 +330,7 @@ export class DateTimePicker extends Component {
         isDateValid: { type: Function, optional: true },
         dayCellClass: { type: Function, optional: true },
         tz: { type: String, optional: true },
+        showSeconds: { type: Boolean, optional: true },
```

Comment:
remove this, it's not relevant for the picker (obviously you'll have a props validation error, see my comments below, I suggest not to add it to pickerProps)